utils/configcheck.py

```diff
@@ -52,6 +56,29 @@ def sd_configcheck(agentConfig):
         configs = load_check_directory(agentConfig, get_hostname(agentConfig))
         get_sd_configcheck(agentConfig, configs)
 
+def agent_container_inspect():
+    print("-- docker_inspect -- \n")
+    dockerutil = DockerUtil()
+    # Self inspection based on cgroups
+
+    proc_path = dockerutil._docker_root + '/proc/self/cgroup'
+    with open(proc_path, 'r') as fp:
+        lines = map(lambda x: x.split(':'), fp.read().splitlines())
+        subsystems = dict(zip(map(lambda x: x[1], lines), map(dockerutil._parse_subsystem, lines)))
+        for k in subsystems.keys():
+            cgroup_regex_list = re.search('(.*/)+(.+?)$', subsystems[k])
```

Comment:
the regex should be a module constant. Also you can precise the group for the container id with `[a-z0-9]{64}`. Length is constant as you can see [here](https://github.com/moby/moby/blob/e23965d620e1655f44d47edd8b5e08c2c384fd03/pkg/stringid/stringid.go#L44-L59)